ads/amoad.js

```diff
@@ -21,13 +21,26 @@ import {loadScript, checkData, validateDataExists} from '../3p/3p';
  * @param {!Object} data
  */
 export function amoad(global, data) {
-  checkData(data, ['sid']);
+  checkData(data, ['sid', 'adType']);
   validateDataExists(data, ['sid']);
 
+  let script;
+  const attrs = {};
+  if (data['adType'] === 'native') {
+    script = 'https://j.amoad.com/js/n.js';
+    attrs['class'] = 'amoad_native';
+    attrs['data-sid'] = data.sid;
+  } else {
+    script = 'https://j.amoad.com/js/a.js';
+    attrs['class'] = `amoad_frame sid_${data.sid} container_div sp`;
+  }
+  global.amoadOption = {ampData: data};
```

Comment:
the string literals in the list will be exempted from being minified. otherwise code might break 
you can test the minified version by running `gulp dist --fortesting` and then open `/examples/ads.amp.min.html`